netlify.toml

```diff
@@ -30,7 +30,7 @@ command = "(env && make web HUGO=$(which hugo) WEBSITE_BASE_URL=${DEPLOY_PRIME_U
     # Force HTTPS only.
     Strict-Transport-Security = "max-age=31536000; includeSubDomains"
     # Load scripts only via HTTPS and from allowed domains.
-    Content-Security-Policy = "default-src https:; script-src https: 'self' 'sha256-3qFt4qPvMCWVUpjUxP5X57GBKae6RHYZ0rMjn9WuNF4='; style-src https: 'self' 'sha256-ouaL9aaaKO9mYd9/M+uyzYAL1CrOXJVRjV2nFgbGI2E='; object-src 'self'"
+    Content-Security-Policy = "default-src https:; script-src https: 'self' 'nonce-93HDK8392dfjh4Gb' 'unsafe-eval' 'sha256-3qFt4qPvMCWVUpjUxP5X57GBKae6RHYZ0rMjn9WuNF4='; style-src https: 'self' 'sha256-ouaL9aaaKO9mYd9/M+uyzYAL1CrOXJVRjV2nFgbGI2E='; object-src 'self'"
```

Comment:
there is still something `unsafe` ;p I imagine it's just easier, but let's never use anything `unsafe` (: What do you think?

Reply:
And if it's necessary let's comment why :hugs:

Reply:
I tried every possible solution I could think of in order to avoid using `unsafe` but none of them was effective. I was able to find a way around using a different approach aside from `unsafe-inline` and I noticed something was missing: **text-to-javascript evaluation**. 
The search box depends on an input string in order to query results and because there was no other way I could do this, I decided to adopt the `unsafe-eval` approach 

During my research, I couldn't find any other way to allow text-to-javascript evaluation in the search box other than unsafe-eval.

Please take a look at this: https://hackerone.com/reports/244724

Reply:
No worries, please try what you can, then we can help when you are blocked :hugs:

Reply:
Okay. Thanks for the feedback @bwplotka